src/ape/contracts/base.py

```diff
@@ -677,6 +677,36 @@ def _mutable_methods_(self) -> Dict[str, ContractTransactionHandler]:
             # NOTE: Must raise AttributeError for __attr__ method or will seg fault
             raise AttributeError(str(err)) from err
 
+    def method_call(self, method_name: str, *args, **kwargs) -> Any:
```

Comment:
I am thinking we may want two methods instead.. One for view calls and one for transactions.
Reason: The return types are really different. Like one returns a `ReceiptAPI` and one returns the return value.

To sound more action-y:

```suggestion
    def call_view_method(self, method_name: str, *args, **kwargs) -> Any:
```

And the one for transactions could be:

```python
def invoke_transaction(self, method_name: str, *args, **kwargs) -> ReceiptAPI:
```